src/elements/content-sidebar/MetadataSidebarRedesignTypes.ts

```diff
@@ -0,0 +1,27 @@
+import API from '../../api';
+import type { WithLoggerProps } from '../../common/types/logging';
+import type { ErrorContextProps } from '../../common/types/api';
+import type { MetadataEditor, MetadataTemplate } from '../../common/types/metadata';
+
+export type ExternalProps = {
+    isFeatureEnabled: boolean;
+    selectedTemplateKey?: string;
```

Comment:
Is it supposed to be part of a props? I believe this will be based on fetched metadata instances for a file, which should be handled by `MetadataSidebarRedesign` component